src/domain/models/models_infra.py

```
@@ -1,6 +1,2 @@
 
-from sqlalchemy.orm import DeclarativeBase
-
-class Base(DeclarativeBase):
-    pass
-
+from src.infrastructure.persistence.models.base import Base
```

Ruff (0.12.2)

`src.infrastructure.persistence.models.base.Base` imported but unused

Remove unused import: `src.infrastructure.persistence.models.base.Base`

(F401)